atomate/vasp/firetasks/glue_tasks.py

```diff
@@ -292,3 +295,77 @@ def run_task(self, fw_spec):
                                "eigenvecs": normalmode_eigenvecs.tolist(),
                                "norms": normalmode_norms.tolist()}
         return FWAction(mod_spec=[{'_set': {'normalmodes': normalmode_dict}}])
+
+
+@explicit_serialize
+class TransferNEBTask(FiretaskBase):
```

Comment:
Currently, all NEB-related tasks are in atomate.vasp.firetasks.neb_tasks. 
It is possible to combine the two tasks but ** TransferNEBTask** is a little bit different:
1) NEB calculation has subfolders
2) If the number of images is not provided by users, this firetask will calculate and pass the value to next firework (NEBFW) and modify "nnodes" accordingly.